binary_search_tree/BinarySearchTree.py

```diff
@@ -0,0 +1,120 @@
+class Node(object):
+    # Node for Binary search Tree and set the Node
+    def __init__(self, data):
+        self.data = data
+        self.left = None
+        self.right = None
+
+# insert data in tree
+def insert(node, data):
+    if node is None:
+        # create a new node
+        node = Node(data)
+        return node
+    else:
+        # go to the left child
+        if data <= node.data:
+            node.left = insert(node.left, data)
+        # got to the right child
+        else:
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmppakk8uf0/binary_search_tree/BinarySearchTree.py
+++ b/tmp/tmppakk8uf0/binary_search_tree/BinarySearchTree.py
@@ -6,6 +6,8 @@
         self.right = None
 
 # insert data in tree
+
+
 def insert(node, data):
     if node is None:
         # create a new node
```